### PR TITLE
Update Azure Marketplace docs version

### PR DESCRIPTION
**Pull request by russcam**
This commit adds 7.11 to the Azure Marketplace docs and sets as current

### CODE REVIEW

**Comment by russcam** [comment]
7.11.1 has been published to the Azure Marketplace, so this PR can be merged anytime

**Review by gtback — APPROVED** [review]
LGTM! After merging we'll want to run a full rebuild of the docs (as we do anytime `current` changes in any of the books).

**Pull request merged**
